pom.xml

```diff
@@ -117,243 +114,160 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
-        <!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>19.0</version>
-        </dependency>
```

Comment:
Removed the Guava dependency in `src/main/java` later in this PR to make this forward compatible with the Guava upgrade in Jenkins 2.332.x.